vllm/attention/selector.py

```diff
@@ -10,6 +11,8 @@
 
 logger = init_logger(__name__)
 
+VLLM_ATTENTION_BACKEND = "VLLM_ATTENTION_BACKEND"
+
 
 class _Backend(enum.Enum):
```

Comment:
i think it's pretty easy to change this to string enum and directly match the env var against this